fMRIVolume/scripts/IntensityNormalization.sh

```diff
@@ -35,18 +36,23 @@ Usage() {
   echo "             [--inscout=<input name for scout image (pre-sat EPI)>]"
   echo "             [--oscout=<output name for normalized scout image>]"
   echo "             [--workingdir=<working dir>]"
+  echo "             [--boldmask=<what mask to use for generating final BOLD timeseries:"
+  echo "                         T1_fMRI_FOV (mask based on T1 and voxels available at all timepoints i.e. FOV - the default)"
```

Comment:
`T1_fMRI_FOV (default): mask based on T1 and voxels available at all timepoints (i.e., the fMRI FOV)`

Reply:
Done.